### PR TITLE
Add chapter03 tutorial, about 'how to use oceanbase'.

### PR DESCRIPTION
**Pull request by meiq4096**
### What changes were proposed in this pull request?

Add chapter02 ：how to use OceanBase

### Will break the compatibility? How if so?

NO.

### How was this patch tested?

Preview it using web browser.

### Checklist

- [ ] I've run the tests to see all new and existing tests pass.
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above.
- [x] I've informed the technical writer about the documentation change if necessary.

### CODE REVIEW

**Review by LINxiansheng — APPROVED** [review]
LGTM

**Comment by LINxiansheng** [comment]
Sorry, we can not merge the pull request.
We do not expect that binary files are exist in the code repository because if the any binary file changes and then the repository size will add the file size.
Could you use some other technology skills to escape the problem for example some online images or git large file storage.

**Review by LINxiansheng — APPROVED** [review]

**Review by yangzhifeng — APPROVED** [review]

File: docs/docs/junior-training/ob-quick-start/chapter03/3.5.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# 如何使用 MySQL 租户做常见数据库开发
+
+## 用户管理
+
+这里用户只租户里的用户，跟传统 MySQL 里的用户概念一样。
```
Comment (Amber1990Zhang) [inline docs/docs/junior-training/ob-quick-start/chapter03/3.5.md:5]: 只->指？

**Review by Amber1990Zhang — APPROVED** [review]

**Review by LINxiansheng — APPROVED** [review]
LGTM

**Comment by LINxiansheng** [comment]
Thanks for your contribution.

**Pull request merged**